crates/image_viewer/src/image_info.rs

```diff
@@ -0,0 +1,147 @@
+use crate::ImageView;
+use anyhow;
+use gpui::{div, Context, Entity, IntoElement, ParentElement, Render, Subscription};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use settings::{Settings, SettingsSources};
+use ui::{prelude::*, Button, LabelSize, Window};
+use workspace::{ItemHandle, StatusItemView, Workspace};
+
+#[derive(Clone, Debug, Serialize, Deserialize, JsonSchema, Default)]
+#[serde(rename_all = "snake_case")]
+pub enum ImageFileSizeUnitType {
+    #[default]
+    Binary,
+    Decimal,
+}
+
+impl Settings for ImageFileSizeUnitType {
+    const KEY: Option<&'static str> = Some("image_file_unit_type");
+
+    type FileContent = Self;
+
+    fn load(
+        sources: SettingsSources<Self::FileContent>,
+        _: &mut App,
+    ) -> Result<Self, anyhow::Error> {
+        sources.json_merge().or_else(|_| Ok(Self::Binary))
+    }
+}
+
+pub struct ImageInfo {
+    width: Option<u32>,
+    height: Option<u32>,
+    file_size: Option<u64>,
+    format: Option<String>,
+    color_type: Option<String>,
+    _observe_active_image: Option<Subscription>,
+}
+
+impl ImageInfo {
+    pub fn new(_workspace: &Workspace, cx: &mut App) -> Self {
+        static INIT: std::sync::Once = std::sync::Once::new();
+        INIT.call_once(|| {
+            ImageFileSizeUnitType::register(cx);
```

Comment:
This should not be in a static. As mentioned previously, it should be added as a line in `image_viewer::init()`

Reply:
true. I guess, I left it that way because I couldn't find `init()`, I thought I'd have to write one myself. But it is already present in `image_viewer` as you mentioned.